website/docs/d/oracle_autonomous_database_regular.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_autonomous_database"
+description: |-
+  Gets information about an existing Autonomous Databases.
+---
+
+# Data Source: azurerm_oracle_autonomous_database
+
+Use this data source to access information about an existing Autonomous Databases.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_autonomous_database" "example" {
+  name                = "existing"
+  resource_group_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_oracle_autonomous_database.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Autonomous Databases.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Autonomous Databases exists.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Autonomous Databases.
+
+* `actual_used_data_storage_size_in_tbs` - The current amount of storage in use for user and system data, in terabytes (TB).
+
+* `allocated_storage_size_in_tbs` - The amount of storage currently allocated for the database tables and billed for, rounded up. When auto-scaling is not enabled, this value is equal to the `dataStorageSizeInTBs` value. You can compare this value to the `actualUsedDataStorageSizeInTBs` value to determine if a manual shrink operation is appropriate for your allocated storage.
+
+* `autonomous_database_id` - The database [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm).
+
+* `available_upgrade_versions` - List of Oracle Database versions available for a database upgrade. If there are no version upgrades available, this list is empty.
+
+* `backup_retention_period_in_days` - Retention period, in days, for backups.
+
+* `character_set` - The character set for the autonomous database. 
+
+* `compute_count` - The compute amount (CPUs) available to the database. Minimum and maximum values depend on the compute model and whether the database is an Autonomous Database Serverless instance or an Autonomous Database on Dedicated Exadata Infrastructure.  For an Autonomous Database Serverless instance, the 'ECPU' compute model requires a minimum value of one, for databases in the elastic resource pool and minimum value of two, otherwise. Required when using the `computeModel` parameter. When using `cpuCoreCount` parameter, it is an error to specify computeCount to a non-null value. Providing `computeModel` and `computeCount` is the preferred method for both OCPU and ECPU.
```

Comment:
computeModel and cpuCoreCount should be  compute_model and cpu_core_count ?

Reply:
Removed as not relevant to ODB@A.